### PR TITLE
Implement str accessor (requires pandas master / 1.2+)

### PR DESCRIPTION
**Pull request by xhochy**
Checklist
* [x] Added a `CHANGELOG.md` entry

### CODE REVIEW

File: benchmarks/benchmarks.py [hunk 1]
```diff
@@ -64,49 +77,58 @@ def time_startswith(self):
...
+
+    def time_startswith_ext_fr(self):
+        self.df_ext["str"].fr_str.startswith("10")
 
     def time_endswith(self):
         self.df["str"].str.endswith("10")
 
     def time_endswith_ext(self):
-        self.df_ext["str"].text.endswith("10")
+        self.df_ext["str"].fr_str.endswith("10")
```
Comment (fhoehle) [inline benchmarks/benchmarks.py:89]:
```suggestion
        self.df_ext["str"].str.endswith("10")
```

File: benchmarks/benchmarks.py [hunk 2]
```diff
@@ -64,49 +77,58 @@ def time_startswith(self):
...
+
+    def time_endswith_ext_fr(self):
+        self.df_ext["str"].str.endswith("10")
```
Comment (fhoehle) [inline benchmarks/benchmarks.py:92]:
```suggestion
        self.df_ext["str"].fr_str.endswith("10")
```

File: benchmarks/benchmarks.py [hunk 3]
```diff
@@ -64,49 +77,58 @@ def time_startswith(self):
...
     def time_zfill_ext(self):
-        self.df_ext["str"].text.zfill(10)
+        self.df_ext["str"].fr_str.zfill(10)
 
     def time_contains_no_regex(self):
         self.df["str"].str.contains("1012102", regex=False)
 
     def time_contains_no_regex_ext(self):
-        self.df_ext["str"].text.contains("1012102", regex=False)
+        self.df_ext["str"].str.contains("1012102", regex=False)
```
Comment (fhoehle) [inline benchmarks/benchmarks.py:110]:
```suggestion
        self.df_ext["str"].fr_str.contains("1012102", regex=False)
```

**Review by fhoehle — CHANGES_REQUESTED** [review]

File: benchmarks/benchmarks.py (hunk 3 above)
Comment (xhochy) [reply benchmarks/benchmarks.py:110]: This one is actually correct, it should be `str`

Comment (fhoehle) [reply benchmarks/benchmarks.py:110]: got it

**Review by fhoehle — APPROVED** [review]

**Pull request merged**